molecule/broker-scale-up/molecule.yml

```diff
@@ -94,18 +94,6 @@ platforms:
     privileged: true
     networks:
       - name: confluent
```

Comment:
why are we removing connect2 and ksql2?

Reply:
Having 5 brokers made this scenario very heavy, and was failing many times due to less memory on the system. Anyways this is a broker-scale-up scenario, so there was not much importance in having >1 `ksql` and `connect`. We already have scenarios for `ksql` and connect scale-up.